hnn_core/viz.py

```diff
@@ -110,7 +110,7 @@ def plot_dipole(dpl, tmin=None, tmax=None, ax=None, layer='agg', decim=None,
 
     ax.ticklabel_format(axis='both', scilimits=(-2, 3))
     ax.set_xlabel('Time (ms)')
-    ax.set_ylabel('Dipole moment')
+    ax.set_ylabel(f'Dipole moment ({dpl[0].units})')
```

Comment:
```suggestion
    ax.set_ylabel(f'Dipole moment (scaled {dpl[0].units})')
```
Regarding the discussion about how to handle units, what if we add 'scaled' in the y-label and then add a label+legend to the figure that specifies the scaling factor. Here's an example snippet of how I've handled it in some of my figures: 
![image](https://user-images.githubusercontent.com/20212206/106322867-809cf400-6244-11eb-8edb-595eb780edd1.png)